L.Map.Sync.js

```diff
@@ -83,19 +85,27 @@
         unsync: function (map) {
             var self = this;
 
+            if (this._cursors) {
+                this._cursors.forEach(function (cursor, indx, _cursors) {
+                    if (cursor === map.cursor) {
+                        _cursors.splice(indx, 1)
```

Comment:
@jieter you are right... but eslint has not detected it (tests are ok). Shouldn't be eslint configuration more strict? (sure, this is for another PR, not now)